src/Files.App/UserControls/NavigationToolbar.xaml

```diff
@@ -383,42 +384,51 @@
 				IconOnActive="{controls:ThemedIconMarkup Style={StaticResource App.ThemedIcons.Omnibar.Commands}, IsFilled=True}"
 				IconOnInactive="{controls:ThemedIconMarkup Style={StaticResource App.ThemedIcons.Omnibar.Commands}, IconType=Outline}"
 				ModeName="{x:Bind Commands.OpenCommandPalette.LabelWithHotKey, Mode=OneWay}"
-				PlaceholderText="{helpers:ResourceString Name=OmnibarCommandPaletteModeTextPlaceholder}">
-				<!--<controls:OmnibarMode.SuggestionItemTemplate>
-					<DataTemplate x:DataType="data:OmnibarPaletteSuggestionItem">
-						<Grid Height="48" ColumnSpacing="12">
+				PlaceholderText="{helpers:ResourceString Name=OmnibarCommandPaletteModeTextPlaceholder}"
+				SuggestionItemsSource="{x:Bind ViewModel.OmnibarCommandPaletteModeSuggestionItems, Mode=OneWay}"
+				Text="{x:Bind ViewModel.OmnibarCommandPaletteModeText, Mode=TwoWay}"
+				TextMemberPath="Text"
+				UpdateTextOnSelect="False">
+				<controls:OmnibarMode.SuggestionItemTemplate>
+					<DataTemplate x:DataType="dataitems:NavigationBarSuggestionItem">
+						<Grid ColumnSpacing="12">
 							<Grid.ColumnDefinitions>
-								<ColumnDefinition Width="Auto" />
+								<ColumnDefinition Width="16" />
 								<ColumnDefinition Width="*" />
 								<ColumnDefinition Width="Auto" />
 							</Grid.ColumnDefinitions>
-							<controls:ThemedIcon
-								Width="20"
-								Height="20"
+
+							<Grid Grid.Column="0">
```

Comment:
Does the ViewBox need to be wrapped in a Grid?

Reply:
Yeah.

```xml
<Grid Grid.Column="0">

    <Viewbox Width="16" Height="16" Visibility="{x:Bind Glyph, Converter={StaticResource NullToVisibilityCollapsedConverter}}">
        <FontIcon Foreground="{ThemeResource App.Theme.IconBaseBrush}" Glyph="{x:Bind Glyph}" />
    </Viewbox>

    <controls:ThemedIcon Style="{x:Bind ThemedIconStyle}" Visibility="{x:Bind ThemedIconStyle, Converter={StaticResource NullToVisibilityCollapsedConverter}}" />

</Grid>
```

Reply:
I missed the fact that it also included the `ThemedIcon` control. Looks good.